README.md

```diff
@@ -84,6 +84,33 @@ This can be run on a regular basis using a scheduler like cron. We recommend an
 - If using cron as a scheduler, [lockrun](http://unixwiz.net/tools/lockrun.html) can be used to prevent multiple instance of the same job running simultaneously.
 - If using a queue-based scheduler like Celery Beat, the `SyncManager` class contains `is_queued` and `is_running` extension points which could be used to implement a lock strategy.
 
+## Auto-sync translations
```

Comment:
We've got one heading here for "Synchronisation" and another one for "Auto-sync translations". Can we clear up the terminology a bit here to explain the difference between "sync" (to RWS) and "auto sync" (translations). Turns out naming things is hard.

I think maybe the terminology "(Auto-)Update Translated Pages" might be better here.

Also maybe clarify that scheduling `sync_rws` is basically essential for this package to do anything, whereas `update_translated_pages` is optional (you might not want this).

Reply:
I agree it is confusing. The main reason I used the word "sync" is because Wagtail Localize allows you to "Sync translated pages". I'll try to make this clearer. 

Also agree on your second point.

Reply:
I've updated the doc for both of these sections.